stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -15,50 +17,54 @@ const SecuritiesTable = () => {
 
   useEffect(() => {
     setIsLoading(true);
-    getSecuritiesData().then(securities => {
-      setIsLoading(false);
-      setSecuritiesData(securities);
-    }).catch(() => {
-      setIsLoading(false);
-    });
+    getSecuritiesData()
+      .then(securities => {
+        setIsLoading(false);
+        setSecuritiesData(securities);
+      })
+      .catch(() => {
+        setIsLoading(false);
```

Comment:
It would be nice to handle errors here, maybe show some kind of error message somewhere if grabbing the securities data fails.

Reply:
Okay, I've added a message now: 

![image](https://user-images.githubusercontent.com/47323296/61286739-5ba87380-a7bb-11e9-8669-a5efaf97543e.png)